company.el

```diff
@@ -2909,10 +2916,10 @@ Delay is determined by `company-tooltip-idle-delay'."
 
 (defvar-local company-preview-overlay nil)
 
-(defun company-preview-show-at-point (pos)
+(defun company-preview-show-at-point (pos &optional common)
```

Comment:
I see..., it seems that passing the exact common part can make the code simplified. OK, I will remodify.